sympy/series/limits.py

```diff
@@ -314,6 +314,8 @@ def set_signs(expr):
                     return r
         else:
             if coeff.has(S.Infinity, S.NegativeInfinity, S.ComplexInfinity):
+                if isinstance(coeff, AccumBounds) and ex == S.Zero:
```

Comment:
AccumBounds should probably be tested before infinities. It looks like also finite ones could be returned as is.

Reply:
Yeah , correct I realized this should be consedered too ! Thanks , I have made the change.